source/extensions/access_loggers/common/file_access_log_impl.cc

```diff
@@ -20,6 +20,13 @@ void FileAccessLog::emitLog(const Http::RequestHeaderMap& request_headers,
                                       stream_info, absl::string_view()));
 }
 
+/* UdpFileAccessLog::UdpFileAccessLog(UdpProxy::UdpProxyFilter& filter,const Filesystem::FilePathAndType& access_log_file_info,
+                             AccessLog::FilterPtr&& filter, Formatter::FormatterPtr&& formatter,
+                             AccessLog::AccessLogManager& log_manager)
```

Comment:
why is this constructor commented out?

Reply:
> why is this constructor commented out?

The implementation code is not complete yet, just for compiling, I will fix it soon.